### PR TITLE
DHCJS haiku

### PR DESCRIPTION
**Pull request by DHCJS**
Appreciate for helping me to contribute in Hacktoberfest.

A warm regards from a self-taught Frontend Developer,

Hartato

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request! There are some changes that need to be made in order to format this correctly. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by DHCJS** [comment]
Hi there, I think I've updated my PR; please kindly recheck it and let me know if it's what you desired.
Thank you!

**Review by ltagliaferri — COMMENTED** [review]
Thanks for your interest in this repository. We would like contributors to provide an original haiku. Please refer to our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) to learn more.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by DHCJS** [comment]
An original haiku means that we should use the exact text for the 5-7-5

Distributed cloud  <br>
Like the octopuses' minds  <br>
Across the network  <br>

**Comment by ltagliaferri** [comment]
Original haiku means a new, original haiku that you wrote.

File: _haikus/2022_First-Open-Source-Project.md
Comment (DHCJS) [inline _haikus/2022_First-Open-Source-Project.md:7]: This is my original Haiku

**Review by DHCJS — COMMENTED** [review]
Hope doing it right

File: _haikus/2022_First-Open-Source-Project.md
Comment (ltagliaferri) [reply _haikus/2022_First-Open-Source-Project.md:7]: The haiku you submitted is a famous one by the poet Basho

Comment (DHCJS) [reply _haikus/2022_First-Open-Source-Project.md:7]: Correct! I just google it

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request, but it does not follow the [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) that are outlined in this repo, so we are closing this PR. If you would like to contribute in the future, please review these guidelines before opening a new pull request.

**Pull request closed without merging**
